root/collection/CollectionEditForm.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import noop from 'lodash/noop';
+import React from 'react';
+
+import {withCatalystContext} from '../context';
+import FormRowSelect from '../components/FormRowSelect';
+import FormRowTextArea from '../components/FormRowTextArea';
+import FormRowTextLong from '../components/FormRowTextLong';
+import FormSubmit from '../components/FormSubmit';
+import FormRowCheckbox from '../components/FormRowCheckbox';
+
+import type {CollectionFormT} from './types';
+
+type Props = {|
+  +$c: CatalystContextT,
+  +collectionTypes: SelectOptionsT,
+  +form: CollectionFormT,
+|};
+
+const CollectionEditForm = ({$c, collectionTypes, form}: Props) => {
+  console.log(collectionTypes);
+  const typeOptions = {
+    grouped: false,
+    options: collectionTypes,
+  };
+
+  return (
+    <form action={$c.req.uri} method="post">
+      <fieldset>
+        <legend>{l('Collection details')}</legend>
+        <FormRowTextLong
+          field={form.field.name}
+          label={l('Name:')}
```

Comment:
`addColon` here and below would be nice since I'm sure we have translations for these without the colon already (probably with too, but)